src/pybind/mgr/mgr_module.py

```diff
@@ -1690,8 +1690,8 @@ def get_latest_unlabeled_counter(self,
                            path: str) -> Dict[str, Union[Tuple[float, int],
                                                          Tuple[float, int, int]]]:
         """
-        Called by the plugin to fetch only the newest performance counter data
-        point for a particular counter on a particular service.
+        Called by the plugin to fetch only the newest performance unlabled counter
+        data point for a particular counter on a particular service.
```

Comment:
the "unlabeled" is misplaced in the sentence

Reply:
fixed! thanks